janino/src/main/java/org/codehaus/janino/UnitCompiler.java

```diff
@@ -11748,7 +11748,7 @@ private int getTargetVersion() {
      * @param opIdx One of {@link #EQ}, {@link #NE}, {@link #LT}, {@link #GE}, {@link #GT} or {@link #LE}
      */
     private void
-    ifNumeric(Locatable locatable, int opIdx, Offset dst) {
+    ifNumeric(Locatable locatable, String op, int opIdx, Offset dst) {

```

Comment:
why is it necessary to have  both `op` and `opIdx`?

Reply:
If this condition is not satisfied, I think that op and opIdx is the same condition. So, in this case, does this condition satisfy? In other words, is `opIdx` updated here?
https://github.com/janino-compiler/janino/blob/8f08f2e90352fd2e46187a6e16d00e658b4cba28/janino/src/main/java/org/codehaus/janino/UnitCompiler.java#L4311

Reply:
I modified it a bit. Please check it again?